tests-functional/tests/test_add_keypair_stored_to_keycard.py

```diff
@@ -0,0 +1,36 @@
+import re
+import pytest
+from clients.api import ApiResponseError
+from resources.constants import user_1, wallet_account_details_derivation, keypair_name
+
+
+@pytest.mark.rpc
+class TestAddKeypairStoredToKeycard:
+
+    @pytest.fixture(autouse=True)
+    def setup_backends(self, backend_new_profile):
+        self.account = backend_new_profile("sender")
+
+    def test_add_keypair_stored_to_keycard_flow(self):
+        add_keypair_resp = self.account.accounts_service.add_keypair_via_seed_phrase(
+            user_1.passphrase, self.account.password, keypair_name, wallet_account_details_derivation
+        )
+
+        key_uid = add_keypair_resp.get("key-uid")
+        self.account.accounts_service.get_keypair_by_key_uid(key_uid)
```

Comment:
In this line we verify that the keypair exists, which is ok, but after that we should verify that keystore files for it also exist.